src/components/FormField/FormFieldSelect.js

```diff
@@ -1,36 +1,74 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import Select from 'react-select';
+import { useField } from 'formik';
 import styles from './FormField.module.scss';
+import mainStyles from '~/assets/styles/main';
+import FormFieldErrorBoundary from '../error-boundaries/FormFieldErrorBoundary';
+import variables from '~/assets/styles/variables';
+
+const mapValues = (values) => (values ? values.map((val) => ({ label: val, value: val })) : []);
 
 const FormFieldSelect = ({
-  label, options, onChange, value,
-}) => (
-  <div className={styles.FormField}>
-    <label>{label}</label>
-    <Select
-      className={styles.selectWrapper}
-      options={options}
-      isMulti
-      onChange={onChange}
-      value={value}
-    />
-  </div>
-);
+  label, options, name, placeholder,
+}) => {
+  const [, meta, helpers] = useField(name);
+  const { value: selectedValues } = meta;
+  const { setValue, setTouched } = helpers;
+
+  const onSelectChange = (newValues) => {
+    setValue(newValues && newValues.length > 0 ? newValues.map(({ value }) => value) : null);
```

Comment:
i'm not sure that you need set value to `null`, maybe you can add `if` condition?:
```js
if (newValues && newValues.length) {
  setValue(newValues.map(({ value }) => value))
}
```

Reply:
`newValues` are not null, this is either `[]` or array of genres as strings. So I removed comparing to null and emptiness